plugins/ScheduledReports/templates/reportParametersScheduledReports.twig

```diff
@@ -31,7 +31,7 @@
             var i, field, fields = ['displayFormat', 'emailMe', 'evolutionGraph', 'additionalEmails'];
             for (i in fields) {
                 field = fields[i];
-                if (report.parameters[field]) {
+                if (field in report.parameters) {
```

Comment:
This check prevented values validating to false from being set to the report itself. This caused the "send to me" value from not being set correctly and saving the default value (yes) instead.